babel.config.js

```diff
@@ -61,6 +61,7 @@ module.exports = {
         'babel-plugin-lodash',
         '@babel/plugin-transform-react-display-name',
         '@babel/plugin-proposal-class-properties',
+        '@babel/plugin-proposal-optional-chaining',
```

Comment:
You just need to bump babel-core and babel preset env and babel runtime (maybe other non-plugin babel packages). The plugin is included in babel version >=7.8.4

Reply:
Yeah, good point. I guess we'll keep it as is for now and remove this in future. What are your thoughts?

Reply:
If you want we can keep it. But I think it's not doing anything at this point.